### PR TITLE
Fixed unexpected fling animation over scrolling content

### PR DESCRIPTION
**Pull request by ASalavei**
Related issue:
https://github.com/JetBrains/compose-multiplatform/issues/3335

### CODE REVIEW

File: compose/ui/ui/src/uikitMain/kotlin/androidx/compose/ui/input/pointer/util/VelocityTracker.uikit.kt [hunk 1]
```diff
@@ -16,9 +16,35 @@
...
+ * Some platforms (e.g. iOS) filter certain gestures during velocity calculation.
  */
-internal actual fun VelocityTracker.shouldUse(event: PointerInputChange): Boolean = event.pressed
\ No newline at end of file
+internal actual fun VelocityTracker1D.shouldUseDataPoints(
+    points: FloatArray,
+    times: FloatArray,
+    count: Int
+): Boolean {
+    if (count == 0) {
```
Comment (MatkovIvan) [inline compose/ui/ui/src/uikitMain/kotlin/androidx/compose/ui/input/pointer/util/VelocityTracker.uikit.kt:35]: Should we add that count <= points.size?

File: compose/ui/ui/src/skikoMain/kotlin/androidx/compose/ui/scene/ComposeScenePointer.skiko.kt [hunk 2]
```diff
@@ -127,7 +127,8 @@ internal fun PointerInputEvent(
             it.type,
             issuesEnterExit = it.type == PointerType.Mouse,
             historical = it.historical,
-            scrollDelta = scrollDelta
+            scrollDelta = scrollDelta,
+            it.position
```
Comment (MatkovIvan) [inline compose/ui/ui/src/skikoMain/kotlin/androidx/compose/ui/scene/ComposeScenePointer.skiko.kt]: Let's use parameter name for this line too

File: compose/ui/ui/src/uikitMain/kotlin/androidx/compose/ui/input/pointer/util/VelocityTracker.uikit.kt (hunk 1 above)
Comment (ASalavei) [reply compose/ui/ui/src/uikitMain/kotlin/androidx/compose/ui/input/pointer/util/VelocityTracker.uikit.kt:35]: To make this logic performant, `points` and `times` has fixed size of `HistorySize`. So count hear indicated number of valid data points

File: compose/ui/ui/src/skikoMain/kotlin/androidx/compose/ui/scene/ComposeScenePointer.skiko.kt (hunk 2 above)
Comment (ASalavei) [reply compose/ui/ui/src/skikoMain/kotlin/androidx/compose/ui/scene/ComposeScenePointer.skiko.kt]: Done

**Review by elijah-semyonov — APPROVED** [review]

**Pull request merged**